modules/developer_manual/pages/core/theming.adoc

```diff
@@ -224,6 +225,124 @@ NOTE: When using custom filetype icons in a custom theme, it is necessary to run
 xref:admin_manual:configuration/mimetypes/index.adoc[mimetypes management].
 
+=== How to Change the Background Image
+
+image::core/theming/2_ow_login.png[Change the background login image in ownCloud]
+
+NOTE: This guide assumes that you are in the folder of your custom theme, for example: `/var/www/owncloud/apps/my_custom_theme/`.
+
+==== How to Change the Login Background Image
+
+. Put the new background image in `core/img/background.jpg`.
+. Change the owner and group permissions of the file to your web server user and group.
+. Update `background-image` in `core/css/styles.css`.
+
+[source,css]
+....
+#body-login {
+     background-image: url("../img/background.jpg"); /* path to image /*
+     background-position: 50% 50%; /* ensure optimal scalability /*
+     background-repeat: no-repeat; /* prevent tiled background /*
+     background-size: cover; /* ensure screen coverage /*
+     text-align: center; /* Center Entitlement text, copyright /*
+     background-color: #000000 !important; /* Fallback for old browsers */
+}
+....
+
+If your image size is 1920px x 1680px, you don't need the lines below the path. 
+However, they ensure optimal positioning and scaling.
+
+image::core/theming/3_cu_login.png[Set a custom login background image in ownCloud]
+
+TIP: You can also change the background color, logo, and slogan.
+
+==== How to Change the Login Background Image to a Color
+
+In `core/css/styles.css`, search for the following CSS code:
+
+[source,css]
+....
+#body-login {
+    background: #745bca; /* Old browsers */
+    background: -moz-linear-gradient(top, #947bea 0%, #745bca 100%);
+}
+....
+
+Then, replace it with the following:
+
+[source,css]
+....
+#body-login {
+    background: rgb(31,9,121);
+    background: linear-gradient(90deg, rgba(31,9,121,1) 38%, rgba(2,0,36,1) 58%);
+}
+....
+
+If you only want one color, replace the existing CSS code with this:
+
+[source,css]
+....
+#body-login {
+    background: rgb(31,9,121);
+}
+....
+
+TIP: If you are not sure what color to pick, {css-gradient-url}[CSS Gradient] may be able to help.
+
+To change the icon, replace the files in `core/img` (`logo.png`, and `logo.svg`) with your icons. 
+The reason for the PNG files is to have a fallback option for older browsers.
+
+If you keep the names - you don't need to change the path in `core/css/styles.css`. 
+If you change the names, adjust `styles.css` file accordingly, as in the following example:
+
+[source,css]
+....
+#header .logo {
+    background-image: url('../img/logo.svg');
+    width: 250px;
+    height: 121px;
+}
+....
+
+=== How to Change the Header
+
+image::core/theming/4_oc_header.png[The default ownCloud header]
+
+The image above is the default ownCloud header.
+
+image::core/theming/5_cu_header.png[Set a custom ownCloud header image]
+
+You can change it to a custom color with a custom logo. 
+Search for body-public `#header` in styles.css.
```

Comment:
Search for `#body-public #header` in `core/css/styles.css`.

Reply:
Fixed. Thanks for spotting that.